ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis.ecf

```diff
@@ -29,14 +21,14 @@ export model=gfs
 ############################################################
 # Load modules
 ############################################################
-module load lsf/${lsf_ver}
-module load impi/${impi_ver}
-module load CFP/${CFP_ver}
-module load python/${python_ver}
-module load NetCDF-parallel/${netcdf_parallel_ver}
-module load HDF5-parallel/${hdf5_parallel_ver}
+module load netcdf/${netcdf_ver}
+module load hdf5/${hdf5_ver}
```

Comment:
This order needs to be reversed.

Reply:
This comment is for everywhere the netcdf and hdf5 modules are loaded.
netcdf depends on hdf5 and the latter needs to be loaded before.

It is likely there are other module loads that are not following their dependency tree.
Reviewing that will take a long time at this point.